topiary/build.rs

```diff
@@ -0,0 +1,74 @@
+use itertools::Itertools;
+use std::collections::HashMap;
+use std::io::Write;
+use std::path::PathBuf;
+use std::{env::current_dir, fs};
+
+fn to_js_string(path: PathBuf) -> String {
+    fs::read_to_string(path)
+        .unwrap()
+        // Escape the escape char itself. Must be done first.
+        .replace('\\', "\\\\")
+        // Backticks are delimiters for template literals in JS.
+        .replace('`', "\\`")
+        // Template literals interpolate `${foo}` as the value of foo.
+        // Escape "${" to prevent this.
+        .replace("${", "\\${")
+}
+
+fn main() {
+    println!("cargo:rerun-if-changed=../languages/");
+    println!("cargo:rerun-if-changed=./tests/samples/input/");
+
+    // Export test samples and queries as JS files
+    let language_dir = current_dir().unwrap().join("../languages/");
```

Comment:
I wonder if we should limit this to the "officially supported" languages? Or at least indicate in the playground whether a language is experimental or not.

Reply:
Maybe we could warn the user, but I don't think it makes sense to limit to the officially supported languages: the playground is just that, a playground. People shouldn't rely on its output for anything

Reply:
I agree; limiting is too strict. Perhaps all that needs to be done is add "experimental" against the appropriate languages in the selector. (Again, I don't think this is needed for this PR.)